js/gulpfile.js

```diff
@@ -325,14 +319,23 @@ gulp.task(
         "ice:bundle",
         "test:common:generate",
         "test:common:bundle",
-        gulp.series(tests.map(testBuildTask)),
-        gulp.series(tests.map(testTypeScriptCompileTask)),
-        gulp.series(tests.map(testBundleTask)),
-        gulp.series(tests.map(testAssetsTask)),
+        gulp.series(tests.map((testName) => testTask(testName, "build"))),
+        gulp.series(tests.map((testName) => testTask(testName, "ts-compile"))),
+        gulp.series(tests.map((testName) => testTask(testName, "bundle"))),
+        gulp.series(tests.map((testName) => testTask(testName, "copy:assets"))),
     ),
 );
 
-gulp.task("test:clean", gulp.parallel("test:common:clean", tests.map(testCleanTask)));
+gulp.task("test:bundle:clean", () => deleteAsync("dist"));
+
+gulp.task(
+    "test:clean",
+    gulp.series(
```

Comment:
Very minor, but why switch the cleaning from parallel to series?
More consistent with the other tasks we define I guess?

I'm not against it, just curious.

Reply:
We got a few out of memory errors in GitHub Action workers